Adyen/CheckoutConfiguration/CheckoutConfiguration.swift

```diff
@@ -6,6 +6,7 @@
 
 import Foundation
 
+// TODO: add description
```

Comment:
A container structure to pass specific payment methods configurations that override default behavior?